traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/Router.java

```diff
@@ -198,6 +198,14 @@ public static AuthenticatedUrlRequest catalog(Session session, CatalogQuery cata
             routeBuilder.param("city", catalogQuery.getCity());
         }
 
+        if (catalogQuery.getLocation() != null) {
+            Double latitude = catalogQuery.getLocation().getLatitude();
+            Double longitude = catalogQuery.getLocation().getLongitude();
+
+            routeBuilder.param("latidude", latitude.toString());
```

Comment:
better to use. String.valueOf(catalogQuery.getLocation().getLatitude()).

not that its wrong but we normally try to avoid using Object classes for primitives. also in traveler base code we always have used String.valueOf(primitive)

Reply:
There is also typo in the key - `latidude` instead of `latitude` based on swagger docs

Reply:
its the latitude of a dude :D